packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart

```diff
@@ -427,6 +436,7 @@ void main() {
   "helloWorldInterpolation": "[{hello}] #{world}#",
   "helloWorldsInterpolation": "{count,plural, other {[{hello}] -{world}- #{count}#}}",
   "singleQuote": "Flutter's amazing!",
+  "singleQuotePlural": "{count,plural, =1{Flutter's amazing, times 1!} other{Flutter's amazing, times {count}!}",
```

Comment:
Once we use `generateString`, we could also do a `doubleQuotePlural` test here

Reply:
Could we move this unrelated small fix to a separate PR?

Reply:
I moved it out to a separate PR here:
https://github.com/flutter/flutter/pull/54185